src/components/MapBox.vue

```diff
@@ -244,6 +246,12 @@ export default {
               map.setLayoutProperty(clickedLayer, "visibility", "none");
               this.className = "";
             } else {
+              if(parent.id === "streams"){
+                for(var i = 0; i < kids.length; i++){
```

Comment:
Can change this to 'let' - not that it will make a functional difference here, but just for consistency sake. Also, I am not sure if 'parent' and 'kid' are the most distinct of names ;), but maybe they do retain a sense of generic-ness.